common/models/user.js

```diff
@@ -556,25 +560,19 @@ module.exports = function(User) {
     // TODO - support more verification types
     function sendEmail(user) {
       verifyOptions.verifyHref += '&token=' + user.verificationToken;
-
       verifyOptions.verificationToken = user.verificationToken;
 
       verifyOptions.text = verifyOptions.text || g.f('Please verify your email by opening ' +
         'this link in a web browser:\n\t%s', verifyOptions.verifyHref);
 
       verifyOptions.text = verifyOptions.text.replace(/\{href\}/g, verifyOptions.verifyHref);
 
```

Comment:
these 4 assignments can't be done before since they depend on user toke generation
we could bring back verifyOptions assertion right after token generation but we'd sacrifice on performance in case of badly defined options